index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Gentium+Plus:wght@700&family=Poppins&family=Roboto:wght@700&display=swap"
+    rel="stylesheet">
+  <script src="https://kit.fontawesome.com/213f6776ae.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="style.css">
+    <title>My portfolio</title>
+</head>
+<body>
+
+    <header>
+        <div class="logo">
+          <a href="" id="logo">Ikonde Arnold Nekemiah</a>
+        </div>
+        <nav class="dropdown">
+          <img src="./Images/Icon - Menu.png" alt="">
+        </nav>
+      </header>
+      <section class="headline">
+        <div class="headline-container">
+          <h1>Hey There. I'm Arnold</h1>
+          <p>
+            <span>I’m a Software Developer</span>
+          </p>
```

Comment:
- [ ] I'm impressed with your design @arnoldnekemiah, It's also great you used an `h1` tag for the main title. But for the subtitle ("I’am a Software Developer"), we should use some heading tag other than `h1`. that can be from `h2` to `h6`.